appinventor/components/src/com/google/appinventor/components/runtime/Map.java

```diff
@@ -350,6 +353,29 @@ public void MapTypeAbstract(MapType type) {
     mapController.setMapTypeAbstract(type);
   }
 
+  /**
+   * @return Returns the custom URL of the base tile layer in use by the map.
+   */
+  @DesignerProperty(editorType = PropertyTypeConstants.PROPERTY_TYPE_TEXT,
+      defaultValue = "https://tile.openstreetmap.org/{z}/{x}/{y}.png")
+  @SimpleProperty(category = PropertyCategory.APPEARANCE,
```

Comment:
Do we want to consider this an ADVANCED property or just APPEARANCE?

Reply:
IMHO, not ADVANCED (I see an example that uses it is ChatBot and ImageBot), and APPEARANCE would be appropriate, especially to make it most accessible for users to easily change.